src/ess/polarization/base.py

```diff
@@ -393,12 +390,22 @@ def __call__(self, time: sc.Variable) -> sc.Variable:
         return self.C * sc.exp(-time / self.T1)
 
 
-def transmission_function(
-    transmission_empty_glass: sc.Variable,
-    opacity: sc.Variable,
-    polarization: sc.Variable,
-) -> sc.Variable:
-    return transmission_empty_glass * sc.exp(-opacity) * sc.cosh(opacity * polarization)
+@dataclass
+class He3TransmissionFunction(Generic[Cell]):
```

Comment:
As shortly discussed yesterday, its dangeroups to call this the same as the transmission function below, which still has to be defined (as it is not the same). This I would call instead He3TransmissionFunctionHalfPolarized, whereas below we should call it He3TransmissionFunctionFullPolarized (see comment below)

Reply:
I think this may be a misunderstanding? This defines a new type (a `class`). And *instance* of this type can be called like a function. We will have various instances of this type (unpolarized, polarized). The names you give would corresponds to variable names of the instances of this type. But they do have the same type (it is the same function, but initialized with different parameters.

Does that make sense?

Reply:
Hm, no it is not really the exact same function. The transmission for an unpolarized incoming beam can not be written with a cosh, that only holds for unpolarized incoming

Reply:
Can you point me to the docs page and section where this is written down?

Reply:
[esspolarization](https://github.com/scipp/esspolarization/tree/main)/[docs](https://github.com/scipp/esspolarization/tree/main/docs)/[user-guide](https://github.com/scipp/esspolarization/tree/main/docs/user-guide)

/sans-polarization-analysis-methodology.ipynb
--> section 'Spin-dependent transmission values'